appengine/flexible/multiple_services/api_gateway/README.md

```diff
@@ -0,0 +1,56 @@
+# Python Google Cloud Microservices Example - API Gateway
+
+This example demonstrates how to deploy multiple python services to [App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/)
+
+## To Run Locally
```

Comment:
This local run story is pretty rough, and some of that blame of course falls on Google. Do you have any ideas on how we can improve this for this sample (procfile + honcho)? Overall?